### PR TITLE
RAFD-3272: data prep, tabs should not be shared among users

### PR DESCRIPTION
**Pull request by zuxqoj**

### CODE REVIEW

File: wrangler-service/src/main/java/co/cask/wrangler/service/common/Constants.java [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+
+    public static class Hive{
+
+        public static final String HIVE_ENABLE_ADDING_SCHEMA = "system.append.hive.schema.enabled";
+
+        public static final String HIVE_REMOVE_TABLENAME_FROM_COLUMN_NAME_CONFIG_KEY = "hive.resultset.use.unique.column.names";
+    }
+
+
+}
```
Comment (goelrajat) [inline wrangler-service/src/main/java/co/cask/wrangler/service/common/Constants.java]: Please use already existing PropertyIds.java file for constants.

File: wrangler-service/src/main/java/co/cask/wrangler/service/adls/ADLSHandler.java [hunk 2]
```diff
@@ -196,9 +198,7 @@ public void loadADLSFile(HttpServiceRequest request, HttpServiceResponder respon
                 return;
             }
 
-            if (Strings.isNullOrEmpty(scope)) {
-                scope = WorkspaceDataset.DEFAULT_SCOPE;
-            }
+            String group = WorkspaceUtils.getScope(scope, request.getHeader(USER_ID_KEY), getContext().getRuntimeArguments());
```
Comment (goelrajat) [inline wrangler-service/src/main/java/co/cask/wrangler/service/adls/ADLSHandler.java]: Please rename the local variable 'group' to something more appropriate eg. 'effectiveScope' as the method which is returning it is 'getScope'

**Review by goelrajat — CHANGES_REQUESTED** [review]

File: wrangler-service/src/main/java/co/cask/wrangler/service/common/Constants.java (hunk 1 above)
Comment (zuxqoj) [reply wrangler-service/src/main/java/co/cask/wrangler/service/common/Constants.java]: done, thanks.

File: wrangler-service/src/main/java/co/cask/wrangler/service/adls/ADLSHandler.java (hunk 2 above)
Comment (zuxqoj) [reply wrangler-service/src/main/java/co/cask/wrangler/service/adls/ADLSHandler.java]: done, thanks.

**Comment by goelrajat** [comment]
LGTM

**Review by goelrajat — APPROVED** [review]

**Pull request merged**